package.json

```diff
@@ -46,56 +46,57 @@
   },
   "dependencies": {
     "algoliasearch-helper": "^2.26.1",
-    "instantsearch.js": "^3.1.1"
+    "core-js": "^2.6.5",
```

Comment:
This likely also isn't necessary, but could be because of babel? We didn't need it before though

Reply:
One of the updated dependencies was requiring the new version of it.